### PR TITLE
Translated README.md in marathi

### PR DESCRIPTION
**Pull request by praj9719**

### CODE REVIEW

**Comment by erictleung** [comment]
@praj9719 thanks for the translation! I'm unable to review your pull request because I don't know Marathi. I will keep this open until someone who knows Marathi is able to review your work before I can merge it.

**Comment by kcoder63** [comment]
can i review it? i know marathi well

**Comment by erictleung** [comment]
@kcoder63 go for it! Take a look at the files here https://github.com/freeCodeCamp/how-to-contribute-to-open-source/pull/499/files and go down to where it says `README-MR.md`. You might have to click on "Load diff" to expand the file because it is too big to show initially. 

To review the text and make suggestions, you can hover and click on the left hand side of the file when you see a plus sigh (`+`) pop up. From there you can make suggestions by pressing <kbd>Ctrl+g</kbd> while in the comment box.

**Comment by kcoder63** [comment]
Thank you , it can take some 2-3 days if that's fine

File: README-MR.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+- [An Intro to Open Source](https://www.digitalocean.com/community/tutorial_series/an-introduction-to-open-source) - येथे गिटहबवर योगदानाच्या यशासाठी आपल्या मार्गदर्शकासाठी डिजिटलऑशन मधील शिकवण्या.
+- [Issuehub.io](http://issuehub.io/) - लेबल आणि भाषेद्वारे गीटहब समस्या शोधण्याचे एक साधन.
+- [Code Triage](https://www.codetriage.com/) - आणखी एक, खरोखर छान, लोकप्रिय भांडार शोधण्याचे साधन आणि भाषेद्वारे फिल्टर केलेले प्रश्न.
+- [Awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners) - नवीन योगदानकर्त्यांसाठी चांगल्या बगसह प्रोजेक्ट्स एकत्रित करणारे आणि त्यांचे वर्णन करण्यासाठी लेबले लागू करणारे एक गीटहब रेपो.
+- [Open Source Guides](https://opensource.guide/) - लोक, समुदाय आणि कंपन्यांना ओपन सोर्स प्रोजेक्टमध्ये कसे चालवायचे आणि योगदान कसे द्यायचे हे शिकू इच्छितात अशा स्त्रोतांचे संग्रह.
+- [45 Github Issues Dos and Don’ts](https://hackernoon.com/45-github-issues-dos-and-donts-dfec9ab4b612) - गिटहब वर करा आणि काय करू नका.
+- [GitHub Guides](https://guides.github.com/) - GitHub प्रभावीपणे कसे वापरावे याबद्दल मूलभूत मार्गदर्शक.
```
Comment (kcoder63) [inline README-MR.md:56]:
```suggestion
- [GitHub Guides](https://guides.github.com/) - गिटहब प्रभावीपणे कसे वापरावे याबद्दल मूलभूत मार्गदर्शक.
```

File: README-MR.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+- [Contribute to Open Source](https://github.com/danthareja/contribute-to-open-source) - सिम्युलेशन प्रोजेक्टमध्ये कोडचे योगदान देऊन गिटहब वर्कफ्लो जाणून घ्या.
+- [Linux Foundation's Open Source Guides for the Enterprise](https://www.linuxfoundation.org/resources/open-source-guides/) - लिनक्स फाऊंडेशनचे ओपन सोर्स प्रकल्पांसाठी मार्गदर्शक.
+- [CSS Tricks An Open Source Etiquette Guidebook](https://css-tricks.com/open-source-etiquette-guidebook/) - केंट सी. डॉड्स आणि सारा ड्रॅसनर यांनी लिहिलेले ओपन सोर्स शिष्टाचार मार्गदर्शक पुस्तिका.
+- [A to Z Resources for Students](https://github.com/dipakkr/A-to-Z-Resources-for-Students) - नवीन कोडींग भाषा शिकण्यासाठी महाविद्यालयीन विद्यार्थ्यांसाठी स्त्रोत आणि संधींची क्युरेटेड यादी.
+- [Pull Request Roulette](http://www.pullrequestroulette.com/) - या साइटवर गीथबवर होस्ट केलेल्या ओपन सोर्स प्रकल्पांच्या पुनरावलोकनासाठी सबमिट केलेल्या पुल विनंत्यांची यादी आहे.
```
Comment (kcoder63) [inline README-MR.md:61]:
```suggestion
- [Pull Request Roulette](http://www.pullrequestroulette.com/) - या साइटवर गिटहब वर होस्ट केलेल्या ओपन सोर्स प्रकल्पांच्या पुनरावलोकनासाठी सबमिट केलेल्या पुल विनंत्यांची यादी आहे.
```

File: README-MR.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+- ["How to get started with Open Source by" Sayan Chowdhury](https://www.hackerearth.com:443/getstarted-opensource/) - या लेखात नवशिक्यांसाठी त्यांच्या आवडत्या भाषेच्या स्वारस्यावर आधारित मुक्त स्त्रोतासाठी योगदान देण्याच्या संसाधनांचा समावेश आहे.
+- ["Browse good first issues to start contributing to open source"](https://github.blog/2020-01-22-browse-good-first-issues-to-start-contributing-to-open-source/) - ओपन सोर्समध्ये योगदान देणे सुरू करण्यासाठी आता गिटहब आपल्याला प्रथम चांगले अंक शोधण्यात मदत करते.
+- ["How to Contribute to Open Source Project" by Maryna Z](https://rubygarage.org/blog/how-contribute-to-open-source-projects) - हा विस्तृत लेख व्यवसायाकडे निर्देशित आहे (परंतु वैयक्तिक योगदानकर्त्यांसाठी अद्याप उपयुक्त आहे) जिथे ते कशासाठी, कसे आणि कोणत्या मुक्त-स्त्रोत प्रकल्पांमध्ये योगदान द्यावे याबद्दल चर्चा केली आहे.
+- ["start-here-guidelines" by Andrei](https://github.com/zero-to-mastery/start-here-guidelines) -
+ओपनसोर्सच्या खेळाच्या मैदानापासून सुरू होण्यास ओपनसोर्सच्या जगात जाऊ दे. विशेषतः शिक्षण आणि व्यावहारिक अनुभवाच्या उद्देशाने डिझाइन केलेले.
+
+## डायरेक्ट गीथब सर्च
```
Comment (kcoder63) [inline README-MR.md:71]:
```suggestion
## डायरेक्ट गिटहब सर्च
```

File: README-MR.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+GitHub वर सहयोग देण्यासाठी थेट योग्य मुद्द्यांकडे निर्देश करणारे दुवे शोधा.
```
Comment (kcoder63) [inline README-MR.md:72]:
```suggestion
गिटहब वर सहयोग देण्यासाठी थेट योग्य मुद्द्यांकडे निर्देश करणारे दुवे शोधा.
```

File: README-MR.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+## वापर आवृत्ती नियंत्रण
+- [Think Like (a) Git](http://think-like-a-git.net/) - "प्रगत नवशिक्यांसाठी" गिट परिचय, परंतु गिटचा सुरक्षितपणे प्रयोग करण्यासाठी आपल्याला एक सोपी रणनीती देण्यासाठी आपण अद्याप धडपडत आहात.
+- [Try Git](https://try.github.io/) - आपल्या ब्राउझरमधून 15 मिनिटांत गिट विनामूल्य वापरा.
+- [Everyday Git](https://git-scm.com/docs/giteveryday) - दररोज गिटसाठी उपयुक्त कमांड्सचा एक सेट.
+- [Oh shit, git!](https://ohshitgit.com/) - साध्या इंग्रजीत वर्णन केलेल्या `git` सामान्य चुकांमधून कसे मुक्त व्हावे; देखील पहा [Dangit, git!](https://dangitgit.com/) शपथ न देता पृष्ठासाठी.
+- [Atlassian Git Tutorials](https://www.atlassian.com/git/tutorials) - `git` वापरण्यावरील विविध ट्यूटोरियल
+- [GitHub Git Cheat Sheet](https://education.github.com/git-cheat-sheet-education.pdf) (PDF)
+- [freeCodeCamp's Wiki on Git Resources](https://forum.freecodecamp.org/t/wiki-git-resources/13136)
+- [GitHub Flow](https://www.youtube.com/watch?v=juLIxo42A_s) (42:06) - पुल विनंती कशी करावी याबद्दल गिटहब चर्चा.
+- [GitHub Learning Resources](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/git-and-github-learning-resources) - Git आणि GitHub शिक्षण संसाधने
```
Comment (kcoder63) [inline README-MR.md:117]:
```suggestion
- [GitHub Learning Resources](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/git-and-github-learning-resources) - गिट आणि गिटहब शिक्षण संसाधने
```

**Review by kcoder63 — CHANGES_REQUESTED** [review]

**Comment by PrajaktaSathe** [comment]
> @praj9719 thanks for the translation! I'm unable to review your pull request because I don't know Marathi. I will keep this open until someone who knows Marathi is able to review your work before I can merge it.

@erictleung 
I know Marathi well...and can check the doc once too! 😄 
Can you assign me this?
Thanks!

**Comment by kcoder63** [comment]
> > @praj9719 thanks for the translation! I'm unable to review your pull request because I don't know Marathi. I will keep this open until someone who knows Marathi is able to review your work before I can merge it.
> 
> @erictleung
> I know Marathi well...and can check the doc once too! 😄
> Can you assign me this?
> Thanks!

I already reviewed it here

File: README-MR.md [hunk 6]
```diff
@@ -22,54 +22,54 @@
...
 
 [![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
 [![first-timers-only Friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://www.firsttimersonly.com/)
 [![Build Status](https://api.travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source.svg?branch=master)](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source)
 
-This is a list of resources for people who are new to contributing to Open Source.
+ओपन सोर्समध्ये सहयोग देण्यासाठी नवीन असलेल्या लोकांच्या संसाधनांची ही सूची आहे.
 
-If you find additional resources, please contribute a pull request.
+आपल्याला अतिरिक्त संसाधने आढळल्यास, कृपया पुल विनंतीस सहयोग द्या.
```
Comment (PrajaktaSathe) [inline README-MR.md:33]: How about - आपल्याला अतिरिक्त संसाधने आढळल्यास, कृपया pull requestनी सहयोग द्या.

File: README-MR.md [hunk 7]
```diff
@@ -22,54 +22,54 @@
...
-- [Mozilla's contributor ecosystem](#mozillas-contributor-ecosystem)
-- [Useful articles for new Open Source contributors](#useful-articles-for-new-open-source-contributors)
-- [Using Version Control](#using-version-control)
-- [Open Source books](#open-source-books)
-- [Open Source contribution initiatives](#open-source-contribution-initiatives)
-- [License](#license)
+- [सर्वसाधारणपणे ओपन सोर्समध्ये योगदान देणे](#सर्वसाधारणपणे-ओपन-सोर्स-स्त्रोत-योगदान-देणे)
+- [डायरेक्ट गिटहब शोध](#डायरेक्ट-गीथब-सर्च)
+- [मोझिलाची योगदानकर्ता पर्यावरणीय प्रणाली](#मोज़िला-योगदानकर्ता-इकोसिस्टम)
+- [नवीन मुक्त स्त्रोत योगदानकर्त्यांसाठी उपयुक्त लेख](#नवीन-मुक्त-स्त्रोत-योगदानकर्त्यांसाठी-उपयुक्त-लेख)
```
Comment (PrajaktaSathe) [inline README-MR.md:42]: I think this would sound better - नवीन open-source योगदानकर्त्यांसाठी उपयुक्त लेख

File: README-MR.md [hunk 8]
```diff
@@ -22,54 +22,54 @@
...
+## सर्वसाधारणपणे ओपन सोर्स स्त्रोत योगदान देणे
+- [The Definitive Guide to Contributing to Open Source](https://www.freecodecamp.org/news/the-definitive-guide-to-contributing-to-open-source-900d5f9f2282/) द्वारा [@DoomHammerNG](https://twitter.com/DoomHammerNG)
+- [An Intro to Open Source](https://www.digitalocean.com/community/tutorial_series/an-introduction-to-open-source) - येथे गिटहबवर योगदानाच्या यशासाठी आपल्या मार्गदर्शकासाठी डिजिटलऑशन मधील शिकवण्या.
+- [Issuehub.io](http://issuehub.io/) - लेबल आणि भाषेद्वारे गीटहब समस्या शोधण्याचे एक साधन.
+- [Code Triage](https://www.codetriage.com/) - आणखी एक, खरोखर छान, लोकप्रिय भांडार शोधण्याचे साधन आणि भाषेद्वारे फिल्टर केलेले प्रश्न.
+- [Awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners) - नवीन योगदानकर्त्यांसाठी चांगल्या बगसह प्रोजेक्ट्स एकत्रित करणारे आणि त्यांचे वर्णन करण्यासाठी लेबले लागू करणारे एक गीटहब रेपो.
+- [Open Source Guides](https://opensource.guide/) - लोक, समुदाय आणि कंपन्यांना ओपन सोर्स प्रोजेक्टमध्ये कसे चालवायचे आणि योगदान कसे द्यायचे हे शिकू इच्छितात अशा स्त्रोतांचे संग्रह.
```
Comment (PrajaktaSathe) [inline README-MR.md:54]: लोक, समुदाय आणि कंपन्यांना ओपन सोर्स प्रोजेक्टमध्ये कसे वापरायचे आणि योगदान कसे द्यायचे हे शिकू इच्छिणाऱ्यांसाठी स्त्रोतांचे संग्रह.

File: README-MR.md [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+## मोज़िला योगदानकर्ता इकोसिस्टम
+- [Good First Bugs](https://bugzil.la/sw:%22[good%20first%20bug]%22&limit=0) - bugs that developers have identified as a good introduction to the project.
+- [Mentored Bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=mentor%3A%40) - bugs that have a mentor assigned who will be there on IRC to help you when you get stuck while working on a fix.
+- [Bugs Ahoy](https://www.joshmatthews.net/bugsahoy/) - a site dedicated to finding bugs on Bugzilla.
+- [Firefox DevTools](http://firefox-dev.tools/) - a site dedicated to bugs filed for the developer tools in the Firefox browser.
+- [What Can I Do For Mozilla](https://whatcanidoformozilla.org/) - figure out what you can work on by answering a bunch of questions about your skill set and interests.
+- [Start Mozilla](https://twitter.com/StartMozilla) - a Twitter account that tweets about issues fit for contributors new to the Mozilla ecosystem.
+
+## नवीन मुक्त स्त्रोत योगदानकर्त्यांसाठी उपयुक्त लेख
+- [How to choose (and contribute to) your first Open Source project](https://github.com/collections/choosing-projects) by [@GitHub](https://github.com/github)
```
Comment (PrajaktaSathe) [inline README-MR.md]: [आपला प्रथम मुक्त स्त्रोत प्रकल्प कसा निवडावा (आणि त्यात सहयोग द्यावा)](https://github.com/collections/choosing-projects) by [@GitHub](https://github.com/github)

File: README-MR.md [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+- [Getting into Open Source for the First Time](https://www.nearform.com/blog/getting-into-open-source-for-the-first-time/) by [@mcdonnelldean](https://github.com/mcdonnelldean)
+- [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/) by [@GitHub](https://github.com/github)
+- [How to Find a Bug in Your Code](https://8thlight.com/blog/doug-bradbury/2016/06/29/how-to-find-bug-in-your-code.html) by [@dougbradbury](https://twitter.com/dougbradbury)
+- [Mastering Markdown](https://guides.github.com/features/mastering-markdown/) by [@GitHub](https://github.com/github)
+- [First mission: Contributors page](https://medium.com/@forCrowd/first-mission-contributors-page-df24e6e70705#.2v2g0no29) by [@forCrowd](https://github.com/forCrowd)
+- [How to make your first Open Source contribution in just 5 minutes](https://www.freecodecamp.org/news/how-to-make-your-first-open-source-contribution-in-just-5-minutes-aaad1fc59c9a/) by [@roshanjossey](https://github.com/Roshanjossey/)
```
Comment (PrajaktaSathe) [inline README-MR.md]:
```suggestion
[केवळ 5 मिनिटांत आपले प्रथम मुक्त स्त्रोत योगदान कसे करावे](https://www.freecodecamp.org/news/how-to-make-your-first-open-source-contribution-in-just-5-minutes-aaad1fc59c9a/) by [@roshanjossey](https://github.com/Roshanjossey/)
```

**Review by erictleung — APPROVED** [review]
LGTM! 👍

**Pull request merged**